README.md

```diff
@@ -10,11 +10,13 @@
 
 This application reminds you to take breaks from the computer, helps you
 remember to stay in touch with and be mindful of your breathing and body
-while sitting at the computer, and helps you concentrate on your in breath
-and out breath when you need a breathing pause.
+while sitting at the computer, and helps you concentrate on breathing
+in and out when you need a breathing pause.
```

Comment:
Nice suggestion here.

I notice this is all one sentence @SunyataZero . @AlphaSoldier would you have any suggestions to break this into two sentences?

Reply:
@andy5995 @AlphaSoldier Yes i also like short sentences, maybe we can find some nice way to split this into parts

Reply:
How's this?

This application reminds you to take breaks from the computer when sitting for long duration of time. It makes you mindful of your breathing by helping you concentrate on breathing in and out when you need a breathing pause.

Reply:
@AlphaSoldier That looks improved. I'll see if I can help tweak it a little further after I get some rest, but maybe you two will have made a final decision on it by then. Gn.

Reply:
@AlphaSoldier I like what you have written, thank you for helping out. Kind Regards, Tord

Reply:
@AlphaSoldier , a ticket for this has been created: #17